modules/govuk/files/node/s_base/amazon-cloudwatch-agent.json

```diff
@@ -13,7 +13,7 @@
           },
           {
             "file_path": "/var/log/secure",
-            "log_group_name": "secure",
+            "log_group_name": "auth-log",
```

Comment:
While we're here, can we fix the file_path on line 15 too? (`secure` => `secure.log`).